docs/configuration.md

```diff
@@ -0,0 +1,199 @@
+
+```python
+    LANGUAGE = 'en'
+```
+
+This will look for the display strings in `languages/en.json`.
+
+If the language strings do need to be changed for a specific project, then any of the keys in the language file can be included in the project configuration file and these will be displayed instead of the strings set in the language file.
+
+## Project Configuration
+
+The project configuration file is written in JSON and has four sections.
+
+1. **behaviourConfiguration** defines the settings to control the behaviour of the system including what links are present in the header
+1. **userFieldsConfiguration** details the questions that the user is asked on the registration page
+1. **websiteTextConfiguration** defines the text that the user sees on the website
+1. **comparisonConfiguration** describes the images being used for the project and how they should be selected for comparison
+
+When the system is started the configuration file requested will be validated and if there are any problems with the file the error message should help to fix them.
+
+The easiest way to create a configuration file is to start with one of the existing examples and modify that to meet the requirements of the project. Each of the sections will be covered below.
+
+**Note**: If the configuration file is edited while the system is running then the system will have to be reset and all
+data in the database will be deleted. Once a system is running do not change a configuration file unless you want to reset the database.
+
+### Behaviour configuration
+
+This section controls aspects of how the system works.
+
+The **exportPathLocation** is required and determines where exported data files are saved when the database is exported.
+
+There is also a required boolean key **renderCookieBanner**, which determines whether the user is asked to accept the website cookies when first opening the website. The title and text for the cookie banner is stored in the language file but it you want to customise it for a particular project then the **siteCookiesTitle** and **siteCookiesText** keys can be added to the project configuration with the strings you want to display.
+
+There are also three booleans which are required to determine which additional pages to include. The configuration of
+these pages is covered in the [Additional pages](#additional-pages) sub-section.
+
+Five further boolean keys are required.
+
+**renderUserItemPreferencePage** determines whether the user is asked to specify whether they know each item in the set of items to be judged. For large item sets it is probably best to determine a users knowledge based on groups rather than individual items and set this value to false. However, it can be used in combination with the group preferences to add an extra level of selection if required. For weighted item configurations this boolean should be set to false.
```

Comment:
```suggestion
**renderUserItemPreferencePage** determines if the user is asked to specify whether they know each item in the set of items to be judged. For large item sets it is probably best to determine a users knowledge based on groups rather than individual items and set this value to false. However, it can be used in combination with the group preferences to add an extra level of selection if required. For weighted item configurations this boolean should be set to false.
```